.goreleaser.yml

```diff
@@ -1,7 +1,7 @@
 # Visit https://goreleaser.com for documentation on how to customize this
 # behavior.
 env:
-  - PROVIDER_VERSION=2.0.0
+  - PROVIDER_VERSION=3.0.0
```

Comment:
The version should be `2.1.0`. This isn't a big enough version for warrant a major upgrade. 
Ideally we align to www.semver.org structure